Formula/libxcb.rb

```diff
@@ -27,11 +26,6 @@ class Libxcb < Formula
     depends_on "graphviz" => :build
   end
 
-  if build.with? "test"
-    depends_on "check" => :build
-    depends_on "libxslt" => [:build, :recommended]
-  end
-
```

Comment:
👎 I'd prefer to keep the option of running `--with-test`.
You can switch `without-test` to `with-test` IMO if tests bother you. I'm also OK with removing/changing `:recommended` option for `libxslt`.